### PR TITLE
Fix letter case

### PR DESCRIPTION
**Pull request by Horay**
When merged this PR will fix letter cases

### CODE REVIEW

File: src/components/text/Text.stories.mdx [hunk 1]
```diff
@@ -25,6 +25,10 @@ import title from "title";
 - `weight`: `boolean` — _optional_
 - `textStyle`: `boolean` — _optional_
 - `align`: `boolean` — _optional_
+- `lineHeight`: `string` — _optional_
+- `width`: `string` — _optional_
+- `nowrap`: `boolean` — _optional_
+- `titleCase`: `boolean` — _optional_
```
Comment (janosio-nordcloud) [inline src/components/text/Text.stories.mdx:31]: ```textTransform: none | capitalize | uppercase | lowercase | initial | inherit``` — _optional_

File: src/components/text/Text.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ type TextProps = {
   lineHeight?: string;
   width?: string;
   nowrap?: boolean;
+  titleCase?: boolean;
   [propName: string]: any;
 };
```
Comment (janosio-nordcloud) [inline src/components/text/Text.tsx:21]: ```textTransform?: none | capitalize | uppercase | lowercase | initial | inherit```

File: src/theme/index.tsx [hunk 3]
```diff
@@ -99,6 +99,7 @@ type Typography = {
   fontWeights: Array<number>;
   fontSizes: Array<string>;
   lineHeight: string;
+  titleCase: string;
```
Comment (janosio-nordcloud) [inline src/theme/index.tsx:102]: "none" | "capitalize" | "uppercase" | "lowercase" | "initial' | "inherit"

File: src/components/input/Input.tsx [hunk 4]
```diff
@@ -69,18 +71,18 @@ export const InputGroup = styled(Flex)<InputGroupProps & SpaceProps>`
...
-      css`
-        ${setStatusColor(status)}
-      `}
-    ${({ popup }) =>
-      popup &&
-      css`
-        cursor: pointer;
-      `}
-    ${space};
+  border-width: ${(props: InputGroupProps) => props.noBorder && "0px"};
```
Comment (nowyDEV) [inline src/components/input/Input.tsx:74]: You don't have to specify units for zero values

**Review by nowyDEV — DISMISSED** [review]

**Review by raczyk — APPROVED** [review]

**Review by janosio-nordcloud — APPROVED** [review]

**Pull request merged**